axelrod/game_params.py

```diff
@@ -4,17 +4,19 @@
 a game (e.g. IPD or ultimatum).
```

Comment:
It may help, going forward, to give a high level description here of what's happening, and what needs to be defined. E.g.: 

A game is specified by the following information:
- A description of positionality (symmetric, asymmetrics, simulaneous play, etc.)
- A definition of the outcomes of an interaction
- A scoring function that takes (...) as parameters and produces a payoff
- etc.

Reply:
Will do